api/can_api_v2_definition.py

```diff
@@ -150,6 +150,24 @@ class Actuals(base_model.APIBaseModel):
  3. We apply an outlier detection filter to the timeseries, which removes any data
     points that seem improbable given recent numbers. Many times this is due to
     backfill of previously unreported cases.
+""",
+    )
+    newDeaths: Optional[int] = pydantic.Field(
+        ...,
+        description="""
+New confirmed or suspected deaths.
+
+New deaths are a processed timeseries of cases - summing new deaths may not equal
```

Comment:
How about "New deaths is an estimate of deaths per day; summing new deaths may not equal the cumulative death count."?